ETC.tsv

```diff
@@ -2707,7 +2707,7 @@ ETC_20150317_002706	Are you sure you want to cancel the item registration?
 ETC_20150317_002707	Your item has been successfully canceled on the Market. Please check that you've received the item.
 ETC_20150317_002708	The equipped item cannot be registered.
 ETC_20150317_002709	Private
-ETC_20150317_002710	Confirm Purchase
+ETC_20150317_002710	Confir
```

Comment:
Exactly, Sometimes github is extremely slow here i don't know why. I just saw it because you saw it. Sorry!